llvm/lib/Transforms/Utils/ProfileVerify.cpp

```diff
@@ -63,6 +67,17 @@ bool ProfileInjector::inject() {
   // will get the same BPI it does if the injector wasn't running.
   auto &BPI = FAM.getResult<BranchProbabilityAnalysis>(F);
 
+  // Inject a function count if there's none. It's reasonable for a pass to
+  // want to clear the MD_prof of a function with zero entry count. From a
+  // metadata economy perspective, if a profile comes empty for a function, it's
```

Comment:
What do you mean by metadata economy? Also should "comes" be "becomes", it changes the meaning a bit.

Reply:
reworded, ptal